src/schema/userSchema.ts

```diff
@@ -0,0 +1,101 @@
+import { Schema } from 'mongoose'
+
+const validateEmail = (email: string) => {
+    const re = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+    return re.test(email)
+}
+
+export const UserSchema: Schema = new Schema({
+    customer_key: {
+        type: String,
+        required: [true, `customer_key is missing.`],
```

Comment:
`` should be replaced with ''. Is linter not throwing error?

Reply:
No, Linter is not throwing error.